app/views/idv/doc_auth/overview.html.erb

```diff
@@ -6,9 +6,9 @@
   } %>
 <% end %>
 
-<h1 class='h3 margin-y-0'><%= t('recover.overview.heading') %></h1>
```

Comment:
multiple h1s!

Reply:
I think this is another one of those hard-to-track disabled-by-default screens which is likely out of sync with what we'd want to have, since we made some changes to these screens recently with doc auth. That being said, as far as short-term, I'm not sure what we'd want to do with it 🤔  

Maybe...

1. Separately, remove the recover flow
2. Separately, reconcile the recover flow overview screens (ideally reusing or adapting existing "welcome" and "agreement" screens to avoid sync issues)
3. Just change the second `h1` to an `h2` since it seems semantically accurate and we'll probably need to revisit these screens again in the future if ever we decide to move forward with enabling the recover flow